src/providers/wms/qgswmsprovider.h

```diff
@@ -219,6 +275,21 @@ class QgsWmsProvider final: public QgsRasterDataProvider
     QList< double > nativeResolutions() const override;
     QgsLayerMetadata layerMetadata() const override;
 
+    // Statitics could be available if the provider has a converter from colors to other value type, the returned statistics depend on the converter
```

Comment:
```suggestion
    // Statistics could be available if the provider has a converter from colors to other value type, the returned statistics depend on the converter
```

Reply:
argh... I had only seen the exclamation point!